web/src/main/java/org/cbioportal/web/GenesetDataController.java

```diff
@@ -0,0 +1,59 @@
+package org.cbioportal.web;
+
+import java.util.List;
+
+import org.cbioportal.model.GenesetGeneticData;
+import org.cbioportal.service.GenesetDataService;
+import org.cbioportal.service.exception.GeneticProfileNotFoundException;
+import org.cbioportal.web.config.annotation.PublicApi;
+import org.cbioportal.web.parameter.GenesetDataFilterCriteria;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+
+@PublicApi
+@RestController
+@Validated
+@Api(tags = "Gene set Data", description = " ")
```

Comment:
As a side note: I think genetic data sounds a bit strange for what it is now in the API. It is actually "alteration data" (not only in the gene set scenario but also for genes). I.e. Currently `GeneticDataController.getAllGeneticDataInSampleInStudy` would be clearer if it would be named `getAllAlterationDataInSampleInStudy`. But this could be a topic of another PR. I agree we should make it at least uniform now. Then it is easier to make the change to another word later.